packages/docs/src/utilities/utils.ts

```diff
@@ -13,3 +13,18 @@ export const readComponent = async (fileName: string): Promise<any> => {
     /* webpackMode: "lazy-once" */
     `../examples/${fileName}.vue`)
 }
+
+/**
+* Find `$t` inside a code and replace it with translation by regex.
+* origin: `... $t('to replace') ...`
+* output: `... translated ...`
+*/
+export function applyTranslations (this: any, code: string) {
+  const replaces = code.match(/(?:\$t)\(.*?\)/) || []
+
+  return replaces.reduce((acc, replaceSource) => {
+    const translation = replaceSource.replace(/(\$t|'|\(|\)|\[\d\])/gi, '')
+
+    return acc.replace(replaceSource, this.$t(translation))
```

Comment:
here we need to replace as `string`, because our `code` prop is bound with `v-bind` directive:
``return acc.replace(replaceSource, `'${this.$t(translation)}'`)``